stko/calculators/torsion_calculators.py

```diff
@@ -153,3 +155,80 @@ def get_results(self, mol):
             generator=self.calculate(mol),
             mol=mol,
         )
+
+
+class MatchedTorsionCalculator(ConstructedMoleculeTorsionCalculator):
+    """
+    Matches rdkit generated torsions with building block torsions.
+    """
+
+    def calculate(self, mol: stk.ConstructedMolecule):
```

Comment:
If we are going to do type hints - we should hint the return type. But for now, the rest of stko does not have type hints (we are slowly doing it in stk!), so maybe leave them out?

Reply:
In that case, you can remove the stk import.